test/SimpleIntervalJob.spec.ts

```
@@ -221,6 +221,33 @@ describe('ToadScheduler', () => {
       scheduler.stop()
     })
 
+    // https://github.com/kibertoad/toad-scheduler/issues/176
+    it('stop() inside an immediate sync run prevents subsequent runs', () => {
+      let counter = 0
+      const scheduler = new ToadScheduler()
+      let job: SimpleIntervalJob
```

ESLint

[error] 'job' is never reassigned. Use 'const' instead.

(prefer-const)